src/Controller/SlackController.php

```diff
@@ -59,10 +59,16 @@ public function slackEndPoint(Request $request): Response
 
         $blocks = $this->slackNotifier->buildDailyBlocks($dailyConferences, $endingCfps);
 
-        $body = [
-            'replace_original' => true,
-            'blocks' => $blocks,
-        ];
+        if (!$blocks) {
+            $body = [
+                'delete_original' => true,
+            ];
+        } else {
+            $body = [
+                'replace_original' => true,
+                'blocks' => $blocks,
+            ];
+        }
```

Comment:
```suggestion
        if ($blocks) {
            $body = [
                'replace_original' => true,
                'blocks' => $blocks,
            ];
        } else {
            $body = [
                'delete_original' => true,
            ];
        }
```

It's easier to read a "positive" condition :)